client/src/components/FilmDetail.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import { useParams } from 'react-router-dom';
+import {useState, useEffect} from 'react';
+
+
+
+
+//const ViewContact = ({contact, toUpdate, toDelete})
+const FilmDetail = () => {
+    const { id } = useParams();
+
+    const [film, setFilm] = useState([]);
+
+
+    //this is the state needed for the UpdateRequest
+    //const [editingcontact, setEditingContact] = useState(null)
+
+    const loadFilmWithID = () => {
+        // A function to fetch the list of students that will be load anytime that list change
+        fetch(`http://localhost:8080/api/films/${id}`)
+            .then((response) => response.json())
+            .then((result) => {
+                console.log("result", result)
```

Comment:
also generally preferable to leave `console.log`, `print()`, etc. out of production code, since the user won't (and usually shouldn't) be seeing it